packages/web/playwright.config.ts

```diff
@@ -0,0 +1,110 @@
+import { defineConfig, devices } from '@playwright/test'
+import fs from 'fs'
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+// require('dotenv').config();
+
+const authFileExists = fs.existsSync('playwright/.auth/user.json')
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+export default defineConfig({
+  testDir: './e2e',
+  /* Run tests in files in parallel */
+  fullyParallel: true,
+  /* Fail the build on CI if you accidentally left test.only in the source code. */
+  forbidOnly: !!process.env.CI,
+  /* Retry on CI only */
+  retries: process.env.CI ? 1 : 0,
+  /* Opt out of parallel tests on CI. */
+  workers: process.env.CI ? 1 : undefined,
+  /* Reporter to use. See https://playwright.dev/docs/test-reporters */
+  reporter: process.env.CI
+    ? [
+        ['junit', { outputFile: 'report.xml', outputDir: 'playwright-report' }],
+        ['html', { open: 'never' }],
+        ['line'],
+        ['blob']
+      ]
+    : 'html',
+  /* Shared settings for all the projects below. See https://playwright.dev/docs/api/class-testoptions. */
+  use: {
+    /* Base URL to use in actions like `await page.goto('/')`. */
+    baseURL: 'http://localhost:3001',
+
+    /* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
+    trace: 'on-first-retry',
```

Comment:
Not sure the implications for CI storage bills, but if possible it would be nice to store traces on every run so that we can debug flaky tests that pass on retry